### ISSUE TITLE
Support of huggingface.co

### ISSUE THREAD

**Issue by rome2rome** [desprit/tg-ai-connector#16]
It's will be great if support https://huggingface.co/. Compute price is 1/2 less than the replicate. 

p.s. sorry I write in the issues section that, the Pull request section doesn't give the to create a new pull.

**Comment by desprit** [comment]
@rome2rome 

Yes, this is doable. Any advice on the network I could run my experiments on? I have never used Hugging Face myself.

**Comment by rome2rome** [comment]
I am a newbie in this matter :). I think - https://huggingface.co/runwayml/stable-diffusion-v1-5 is a great choice. There is a free Hosted inference API, but not sure if there is for remote :) You can generate a token, but I have no chase to try it. 

It's look like to me is the same idea and structure like Replicate , but Nvidia A100 40GB GPU costs ~8$ in Replicate, and only Hugging Face is $4.13.
